SConstruct

```diff
@@ -858,6 +858,8 @@ else:  # GCC, Clang
             common_warnings += ["-Wno-type-limits"]
         if cc_version_major >= 12:  # False positives in our error macros, see GH-58747.
             common_warnings += ["-Wno-return-type"]
+        if cc_version_major < 13:  # `region` & `endregion` not recognized as valid pragmas.
+            common_warnings += ["-Wno-unknown-pragmas"]
```

Comment:
This seems to only be a problem for `directx_headers`. Maybe we should pass it only to the `drivers/d3d12/SCsub`?

Reply:
That's a much better idea, yeah.